input/pagecontent/4-2-syncconsiderations.md

```diff
@@ -156,8 +167,6 @@ Negligible risk. Per typical application security best practices, applications s
 5. User hibernates application A, walks away from workstation, application B remains open and unsecured.
 Negligible risk. Per typical application security best practices, application should automatically secure following a period of un-use.
 
-
-
 ### Re-establishing sync

```

Comment:
Should we include the "standard" note for the Re-establishing sync section that implementor feedback is requested.

Reply:
I think we're going to need to work towards removing some of the "implementer feedback" statements.